DFC.Api.Content/host.json

```diff
@@ -1,3 +1,8 @@
 {
-    "version": "3.0"
+  "version": "2.0",

```

Comment:
Does this and `<AzureFunctionsVersion>v3</AzureFunctionsVersion>` need to tally up?

Reply:
Changing this bit to 3.0 makes the function throw a 503

Reply:
https://github.com/Azure/azure-functions-host/issues/5310 - looks like host.json is the schema version

Reply:
Ah, certainly confused me 😄!